tests/python/gpu/test_operator_gpu.py

```diff
@@ -2331,6 +2331,82 @@ def test_math():
             for op in ops:
                 run_math(op, shape, dtype, check_value=check_value)
 
+
+@with_seed()
+@use_np
+def test_np_arctan2():
+    class TestArctan2(HybridBlock):
+        def __init__(self):
+            super(TestArctan2, self).__init__()
+
+        def hybrid_forward(self, F, x1, x2):
+            return F.np.arctan2(x1, x2)
+
+    #Reduce dimension of src to dimention of des.
```

Comment:
`# Reduce dimension of src to dimension of des.`

Reply:
Done.